xdsl/ir/core.py

```diff
@@ -2147,7 +2151,7 @@ def __getitem__(self, idx: int | slice) -> Block | Sequence[Block]:
 
     def __setitem__(self, idx: int, successor: Block) -> None:
         successors = self._op._successors  # pyright: ignore[reportPrivateUsage]
-        successor_uses = self._op._successor_uses  # pyright: ignore[reportPrivateUsage]
+        successor_uses = self._op.successor_uses
```

Comment:
Or should internals access `_successor_uses` directly?

Reply:
In this case I'd go for private